src/app/zap-templates/zcl/data-model/silabs/ami.xml

```diff
@@ -201,8 +201,8 @@ limitations under the License.
     <item value="0xA2" name="BatteryCoverRemoved"/>
     <item value="0xA3" name="BatteryCoverClosed"/>
     <item value="0xA4" name="ExcessFlow"/>
-    <item value="0xC0" name="CreditOk"/>
-    <item value="0xC1" name="LowCredit"/>
+    <item value="0xB0" name="CreditOk"/>
+    <item value="0xB1" name="LowCredit"/>
```

Comment:
Where is this change coming from?

Reply:
so I see duplicate 0xC0 0XC1
    <item value="0xC0" name="CreditOk"/>
    <item value="0xC1" name="LowCredit"/>
    <item value="0xC0" name="EmergencyCreditInUse"/>
    <item value="0xC1" name="EmergencyCreditExhausted"/>

Then i modify them as b0 and b1

Reply:
But these are coming from some spec.  What does that spec say?

Reply:
actually i cannot find this ami and enums in spec......

Reply:
They're not in the _Matter_ spec.  They're in the Zigbee specs...

Reply:
just search zigbee spec, and have not yet found things related to this one. But technically it is not right here. thanks